src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse.cs

```diff
@@ -17,372 +17,489 @@ public static class Sse
+        ///   CMPSS xmm, xmm/m32, imm8(5)
         /// </summary>
         public static Vector128<float> CompareNotLessThanScalar(Vector128<float> left, Vector128<float> right) => CompareNotLessThanScalar(left, right);
 
         /// <summary>
         /// __m128 _mm_cmpnle_ps (__m128 a,  __m128 b)
+        ///   CMPPS xmm, xmm/m128, imm8(6)
         /// </summary>
         public static Vector128<float> CompareNotLessThanOrEqual(Vector128<float> left, Vector128<float> right) => CompareNotLessThanOrEqual(left, right);
 
         /// <summary>
         /// __m128 _mm_cmpnle_ss (__m128 a,  __m128 b)
+        ///   CMPSS xmm, xmm/m32, imm8(6)
         /// </summary>
         public static Vector128<float> CompareNotLessThanOrEqualScalar(Vector128<float> left, Vector128<float> right) => CompareNotLessThanOrEqualScalar(left, right);
 
         /// <summary>
         /// __m128 _mm_cmpord_ps (__m128 a,  __m128 b)
+        ///   CMPPS xmm, xmm/m128, imm8(7)
         /// </summary>
         public static Vector128<float> CompareOrdered(Vector128<float> left, Vector128<float> right) => CompareOrdered(left, right);
 
         /// <summary>
         /// __m128 _mm_cmpord_ss (__m128 a,  __m128 b)
+        ///   CMPSS xmm, xmm/m32, imm8(7)
         /// </summary>
         public static Vector128<float> CompareOrderedScalar(Vector128<float> left, Vector128<float> right) => CompareOrderedScalar(left, right);
 
         /// <summary>
         /// __m128 _mm_cmpunord_ps (__m128 a,  __m128 b)
+        ///   CMPPS xmm, xmm/m128, imm8(3)
         /// </summary>
         public static Vector128<float> CompareUnordered(Vector128<float> left, Vector128<float> right) => CompareUnordered(left, right);
 
         /// <summary>
         /// __m128 _mm_cmpunord_ss (__m128 a,  __m128 b)
+        ///   CMPSS xmm, xmm/m32, imm8(3)
         /// </summary>
         public static Vector128<float> CompareUnorderedScalar(Vector128<float> left, Vector128<float> right) => CompareUnorderedScalar(left, right);
 
         /// <summary>
         /// int _mm_cvtss_si32 (__m128 a)
+        ///   CVTSS2SI r32, xmm/m32
         /// </summary>
         public static int ConvertToInt32(Vector128<float> value) => ConvertToInt32(value);
         /// <summary>
         /// __int64 _mm_cvtss_si64 (__m128 a)
+        ///   CVTSS2SI r64, xmm/m32
         /// </summary>
         public static long ConvertToInt64(Vector128<float> value) => ConvertToInt64(value);
+
         /// <summary>
         /// float _mm_cvtss_f32 (__m128 a)
+        ///   HELPER: MOVSS
         /// </summary>
         public static float ConvertToSingle(Vector128<float> value) => ConvertToSingle(value);
 
         /// <summary>
         /// __m128 _mm_cvtsi32_ss (__m128 a, int b)
+        ///   CVTSI2SS xmm, reg/m32
         /// </summary>
         public static Vector128<float> ConvertToVector128SingleScalar(Vector128<float> upper, int value) => ConvertToVector128SingleScalar(upper, value);
```

Comment:
`ConvertScalarToVector128Single` might be a better name. Thoughts?

Reply:
Agree.